packages/terra-popup/src/terra-dev-site/test/popup/OverlayPopup.test.jsx

```diff
@@ -11,13 +11,10 @@ class DefaultPopup extends React.Component {
     this.state = { open: false };
   }
 
-  componentWillMount() {
```

Comment:
This was originally using componentWillMount to ensure the overflow was turned off before the popup was positioned, this was mainly an issue with body scrolling.  We'll likely need to move this to render or something.  Similar to what the other overlay uses. Or perhaps switch to the terra-overlay.